### PR TITLE
Nut scanner deps

### PR DESCRIPTION
**Pull request by jimklimov**
Avoid rebuilding nut-scanner upon every invokation of `make` or after `make deps`.

### CODE REVIEW

**Comment by jimklimov** [comment]
Seems there are many callers of this script and/or make target, and in many different contexts (`autogen`, `make all`, `make nut-scanner`, `make dist`, `make distcheck` ...) fixing one breaks another :\

It's complicated (C)

File: tools/Makefile.am [hunk 1]
```diff
@@ -19,55 +19,85 @@ SUBDIRS = . nut-scanner
...
+
+# BSD devd output file
+GENERATED_USB_FILES += ../scripts/devd/nut-usb.conf.in
+
+# UPower output file
+GENERATED_USB_FILES += ../scripts/upower/95-upower-hid.rules
+
+all: nut-scanner-deps $(GENERATED_USB_FILES)
 
 # XXX these rules are all bogus!  They cause un-named target files to
```
Comment (clepple) [inline tools/Makefile.am]: it looks like this comment is out-of-date, at least for this particular rule.

Comment (clepple) [reply tools/Makefile.am]: The way this is rendered is a bit misleading - I meant that "XXX these rules are all bogus!" was out-of-date with respect to the rule below (which does have specific dependencies), not the green portion of the previous diff.

Comment (jimklimov) [reply tools/Makefile.am]: Thanks for pointing out that I never got to it, I hoped to but forgot ;)

**Pull request merged**